docs/src/user/guide/forgot-password.md

```diff
@@ -0,0 +1,33 @@
+# miniSASS Forgot Password Page User Guide
+
+Welcome to the miniSASS Forgot Password Guide. In this guide, I'll walk you through the steps to reset your password if you've forgotten it.
+
+## How do I reset password
+
+Click on the 1️⃣ `click here` link, to go to the forgot password page.
+
+![login form](./img/forgot-password-1.png)
+
+Upon clicking on the link you will be redirected to the forgot password page where you need to enter your email address in the 1️⃣ email field in order to receive a reset password email. Once you entered your email click on the 2️⃣ `Send reset link` button to submit your forgot password request.
+
+![forgot password form](./img/forgot-password-2.png)
+
+On successful submission the success message will be displayed.
+
+![forgot password form](./img/forgot-password-3.png)
+
+Upon initiating the password reset process, you will receive an email containing a link to reset your password. You have three options to access the password update page:
+
+Within the email, there will be a 1️⃣ button labeled `Reset Password`. You can click on this button to be directed to the password update page. Alternatively, you can simply click on the 2️⃣ `Click Here` link provided in the email. This will also take you to the password update page. Or you can manually copy the provided link 3️⃣ from the email and paste it into your web browser's address bar. Press Enter to navigate to the password update page.
```

Comment:
```suggestion
Within the email, there will be a 1️⃣ button labelled `Reset Password`. You can click on this button to be directed to the password update page. Alternatively, you can simply click on the 2️⃣ `Click Here` link provided in the email. This will also take you to the password update page. Or you can manually copy the provided link 3️⃣ from the email and paste it into your web browser's address bar. Press Enter to navigate to the password update page.
```